create_lmodsitepackage.py

```diff
@@ -19,6 +19,18 @@
     return content
 end
 
+local function from_eessi_prefix(t)
+    local eessi_prefix = os.getenv("EESSI_PREFIX")
```

Comment:
In https://github.com/EESSI/software-layer/pull/371 we add the ability to extend EESSI both system-wide (via `host_injections`) and for the individual user (via installation in their home directory), you would still want the hook to run for these cases.

The `host_injections` case should be straightforward:
```
string.gsub(os.getenv("EESSI_PREFIX"), 'versions', 'host_injections')
```

The `$HOME` case is a bit trickier, as we need to decide on a location. It could look like:
```
string.gsub(os.getenv("EESSI_PREFIX"), os.getenv("EESSI_CVMFS_REPO"), pathJoin(os.getenv("HOME"), "eessi"))
```

Reply:
Ok, changed my mind: I just put it in, with a remark that we may need to update the paths depending on what gets chosen in #371.

Reply:
Great, thanks.